packages/amplify-graphql-schema-generator/src/datasource-adapter/datasource-adapter.ts

```diff
@@ -11,14 +12,15 @@ export abstract class DataSourceAdapter {
 
   public abstract getIndexes(tableName: string): Promise<Index[]>;
 
-  public abstract mapDataType(datatype: string, nullable: boolean, tableName: string, fieldName: string, columnType: string): FieldType;
-
   public abstract initialize(): Promise<void>;
 
   public abstract cleanup(): void;
 
   public abstract test(): Promise<boolean>;
 
+  // todo correct function signature
```

Comment:
What's incorrect about it?

Reply:
Nothing is wrong. I forgot to remove the comment after updating.